theano/gpuarray/nerv.py

```diff
@@ -157,8 +157,8 @@ def local_gpua_dot_to_gemm16(op, ctx_name, inputs, outputs):
     B = inputs[1]
     if (A.ndim == 2 and B.ndim == 2 and
             A.dtype == 'float16' and B.dtype == 'float16'):
-        fgraph = getattr(outputs[0], 'fgraph', None)
-        C = gpu_alloc_empty(ctx_name, dtype='float16')(
+        fgraph = inputs[0].fgraph
```

Comment:
remove the change to this line. I don't think it is good. inputs don't always have an fgraph attribute I think.

@Sentient07 do you remember why this line was changed? If not, we should undo it.